src/Synth/LFO.cpp

```diff
@@ -28,8 +28,9 @@ LFO::LFO(const LFOParams &lfopars, float basefreq, const AbsTime &t, WatchManage
     fadeInTime(t, lfopars.delay+lfopars.fadein), //0..8 sec
     waveShape(lfopars.PLFOtype),
     deterministic(!lfopars.Pfreqrand),
-    dt_(t.dt()),
-    lfopars_(lfopars), basefreq_(basefreq),
+        dt_(t.dt()),
+    lfopars_(lfopars), 
+    basefreq_(basefreq),
```

Comment:
These 2/3 lines should not be changed at all, right? They are not related to this PR.

Reply:
it's only code formatting for better readability.